### PR TITLE
Covidcast-indicators version logging and workflow

### LINKED ISSUE #1702: Indicators should print their name and version information on start
One of the common failure modes of our production system is "hey, this widget was supposed to have been updated, but seems to be running old code". To help make this problem easier & quicker to diagnose, we should modify each indicator so that when it starts up, it prints its name and some kind of version information.

Version information could be:
* repo version (this is available in [.bumpversion.cfg](https://github.com/cmu-delphi/covidcast-indicators/blob/main/.bumpversion.cfg) but probably doesn't survive our deployment mechanism)
* indicator version (which we currently don't track, but maybe we should? it would probably live in [setup.py (claims_hosp example)](https://github.com/cmu-delphi/covidcast-indicators/blob/2ec89216a520d5ad1ecbdea7c566b75a63a0024c/claims_hosp/setup.py#L18))
* gitref (not visible in deployed code but we could make jenkins write it down)
* build date (another thing jenkins would have to do)

### PR DESCRIPTION
**Pull request by M5Skid**
### Description
Adds logging for indicator name and covidcast-indicators repository version when started by delphi_utils.runner. Workflow populates version.cfg in indicator directories with current version (same as .bumpversion.cfg), and runner will check this file on individual indicator startup.
- Removed decommissioned indicators from the workflow 

### Changelog

create-release.yml
- copy covidcast-indicators version to indicator directories

runner.py
- check version.cfg for version info, output indicator name and version

### Fixes 
- Fixes #1702

### CODE REVIEW

**Review by krivard — CHANGES_REQUESTED** [review]
Great first pass! This functionality seems like something we should include in unit tests, which would require some refactoring.

We should fix the file I/O before merging but leave tests as future work; I'll open an issue for that.

File: _delphi_utils_python/delphi_utils/runner.py [hunk 1]
```diff
@@ -40,6 +41,21 @@ def run_indicator_pipeline(indicator_fn:  Callable[[Params], None],
...
+    #Get version and indicator name for startup
+    ind_name = indicator_fn.__module__.replace(".run","")
+    #Check for version.cfg in indicator directory
+    if os.path.exists("version.cfg"):
+        ver_file = open("version.cfg")
+        current_version = "not found"
+        for line in ver_file:
+            if "current_version" in line:
+                current_version = str.strip(line)
+                current_version = current_version.replace("current_version = ", "")
```
[the remark this answers is not in the document]
Comment (M5Skid) [reply _delphi_utils_python/delphi_utils/runner.py:54]: Thanks for the additional resources! I think this is a good correction.

**Review by krivard — APPROVED** [review]
👍

**Comment by krivard** [comment]
Note to self: this affects Runner, so there's a tiny but nonzero chance of hosing our entire stable of indicators. To be safe, merge this on a tuesday evening, then wednesday make the staging server run all the indicators and make sure there are no mishaps before releasing to prod.

**Comment by korlaxxalrok** [comment]
@krivard @M5Skid

I think we called this out earlier as an eventual necessity:

Because the new indicator versioning process is happening in the release workflow that ultimately generates a PR against prod, we are going to need to set up an additional build+deploy step in Jenkins to create a new set of indicator targzs built from prod that will contain the version file.

**Review by korlaxxalrok — APPROVED** [review]
Looks good 👍 💥

**Comment by krivard** [comment]
Oh shoot you're failing the linter check in runner.py -- take a look? I'll be able to pick it up an hour to fix and merge if you haven't found it by then.

**Comment by M5Skid** [comment]
It looks like linter is having issues with some white spaces. I took the spaces out and will try to run a quick local test with linter before resubmitting the PR.

**Pull request merged**